### PR TITLE
create resilient directory rm

### PR DESCRIPTION
**Pull request by jperez999**
create resilient remove of packages to support previous and future version installs

### CODE REVIEW

File: docker/dockerfile.torch [hunk 1]
```diff
@@ -22,8 +22,8 @@ RUN apt update -y --fix-missing && \
 
 # Torch Metrics (without torch)
 RUN pip install --no-cache-dir --no-deps torch torchmetrics \
-    && rm -r /usr/local/lib/python3.8/dist-packages/torch \
-    && rm -r /usr/local/lib/python3.8/dist-packages/caffe2
+    && if [ -d /usr/local/lib/python3.8/dist-packages/torch ]; then rm -r /usr/local/lib/python3.8/dist-packages/torch; fi \
```
Comment (mikemckiernan) [inline docker/dockerfile.torch:25]: Drive-by comment: I think you can `rm -rf anything` and A) it'll recurse and B) it never errors, so you can skip the directory check.

```
rm -rf blah

echo $?
0
```

Comment (jperez999) [reply docker/dockerfile.torch:25]: Nice will revamp PR. Good catch.

**Review by mikemckiernan — APPROVED** [review]
lgtm
